Kiosk/App/Networking/ArtsyAPI.swift

```diff
@@ -28,6 +28,7 @@ enum ArtsyAPI {
     
     case MyBiddersForAuction(auctionID: String)
     case MyBidPositionsForAuctionArtwork(auctionID: String, artworkID: String)
+    case MyBidPosition(id: String)
```

Comment:
Ah, that actually makes sense. If the id belongs tok the bid position, let's keep this as-is.